src/sql/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM debian:jessie
+
+# Install postgresql.org APT source
+RUN apt-get update && apt-get install -y \
+  ca-certificates \
+  curl \
+  gnupg \
+  lsb-release \
+  && apt-get clean
+RUN curl -s https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+
+# Install PostgreSQL and packages needed to build
+# PostGIS from source
+RUN apt-get update && apt-get install -y \
+  postgresql-8.4 \
+  postgresql-server-dev-8.4 \
+  build-essential autoconf libtool libcunit1-dev \
+  xsltproc docbook-xsl docbook-mathml \
+  libproj-dev libjson0-dev libxml2-dev \
+  flex bison unzip imagemagick \
+  libgeos-dev && apt-get clean
+
+# Install PostGIS 1.5.8 from source
+RUN curl -s -L \
+  https://ftp.osuosl.org/pub/osgeo/download/postgis/source/postgis-1.5.8.tar.gz | \
+  tar xz && cd postgis-1.5.8 && \
+  ./configure && make && make install && \
+  cd .. && rm postgis-1.5.8 -rf
+
+# Install gazeteer schema
+ADD *.sql build_gazetteer_export.py install.sh /sql/
```

Comment:
%psql% -f gazetteer_web_schema.sql  needs to be added to install.sh to deployed the gazetteer_web schema

Reply:
I had Dockerfile run that script outside of install.sh, see https://github.com/linz/gazetteer/pull/64/files#diff-cc0a840aa5e9e81d98c2b58dd2f7c98eR44 (install_web.sh sources gazetteer_web_schema.sql)